src/Microsoft.OpenApi.Hidi/OpenApiService.cs

```diff
@@ -251,13 +251,13 @@ private static async Task<Stream> GetStream(string input, ILogger logger)
             {
                 try
                 {
-                    using var httpClientHandler = new HttpClientHandler()
+                    var httpClientHandler = new HttpClientHandler()
```

Comment:
Because HttpClient disposes of HttpMessageHandler by default.  You need to pass an explicit parameter into HttpClient to tell it not to.  By using using you would try and dispose it twice.